onsset/onsset.py

```diff
@@ -203,9 +207,10 @@ def set_default_values(cls, base_year, start_year, end_year, hv_line_type=69, hv
         cls.load_moment = load_moment  # for 50mm aluminum conductor under 5% voltage drop (kW m)
 
     def get_lcoe(self, energy_per_cell, people, num_people_per_hh, start_year, end_year, new_connections,
-                 total_energy_per_cell, prev_code, grid_cell_area, additional_mv_line_length=0.0,
-                 capacity_factor=0.9, grid_penalty_ratio=1, fuel_cost=0, elec_loop=0, productive_nodes=0,
-                 additional_transformer=0, penalty=1, get_max_dist=False):
+                 total_energy_per_cell, prev_code, grid_cell_area, mg_diesel_calc, unmet_demand=0, additional_mv_line_length=0.0,
```

Comment:
Perhaps set mg_diesel_calc={} as default, so that this does not have to be passed in for the off-grid technologies later where it is not used, to avoid confusion.